### ISSUE TITLE
Better UI for 'Genres' and Landing page

### ISSUE THREAD

**Issue by Greeshma2903** [Praful932/Kitabe#62]
**Describe the bug**
The carousels used under 'Read Genre Wise' overflows beyond 3 (as indicated ). Also this way it becomes hard/tricky for the user to select a particular Genre (as they have to keep searching).  

Also we can rescale the image below navbar, to redirect the attention to the 'Genres' section.

**To Reproduce**
1. Go to 'https://kitabe-app.herokuapp.com/'
2. Scroll down to 'Read Genre Wise'
4. Click on "next" arrow under any carousel
5. See error

**Expected behavior**
Expected to show me a list of similar Genres.

**Screenshots**
![image](https://user-images.githubusercontent.com/70336930/110748501-ceeedb00-8265-11eb-9f77-cc3667122f94.png)
![image](https://user-images.githubusercontent.com/70336930/110748529-d9a97000-8265-11eb-9b69-c36e9635c713.png)

**Are you willing to contribute to this issue?**
Yes
I have made a mock-up design  here on how we can make the carousel part better :https://www.figma.com/file/quY3GKiD5RgWhsJAg2PKyE/Carousel-draft-Kitaabe?node-id=0%3A1
Being a book lover, I would love to work on this issue, as a _GSSoC'21 participant_.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
I feel the colour theme of the website can be made more vibrant/ cozy (to give modern day library vibes😄). 
My design-mockup is just a suggestion. Feedback is welcome! 
@maneprajakta @Praful932

**Comment by Praful932** [comment]
Hi @Greeshma2903 Thanks for your interest in the project :)
>Expected to show me a list of similar Genres.

Could you elaborate this?
In the mockup could you add how it would look when scaled to a mobile view? particulary the 1st one
For the 1st one it would be nice to have a one block auto scroll
As for the color scheme theres an ongoing issue #48 
Check this out @maneprajakta

**Comment by maneprajakta** [comment]
@Greeshma2903 From the mockup what I could see is that instead of having four carousels in the genre section it should be replaced by a single carousel seems great to me.
Yes, think for mobile view also.

**Comment by Greeshma2903** [comment]
Thank you assigning this to me😊!
Yes sure, I'll draft the interface for small screens and get back to you.

Regarding the expected behavior:
I expected each carousel block to show me a list of similar genres, which it did not. And moreover the genres where more than 3 (as indicated), which might confuse the user.
![Carousel slide indicator](https://user-images.githubusercontent.com/70336930/110889536-51ce6f00-8314-11eb-915d-0bc5669ace54.PNG)

I am sorry but I couldn't understand, the 1st one means the first carousel block right?

**Comment by Greeshma2903** [comment]
> 
> 
> @Greeshma2903 From the mockup what I could see is that instead of having four carousels in the genre section it should be replaced by a single carousel seems great to me.
> Yes, think for mobile view also.

Yes ma'am, working on it😄

**Comment by Praful932** [comment]
@Greeshma2903 not sure about what you meant by **similar genres** 
First one as in this one
![image](https://user-images.githubusercontent.com/45713796/110892304-d1ab0800-8319-11eb-919c-9238fadb3a11.png)
Just call me @Praful932 without any honorifics 😄 This is Open source

**Comment by Greeshma2903** [comment]
Um, actually I thought there were multiple carousels, so each carousel might be a group of similar genres (like fiction and its sub-genres). But that wasn't the case 😅.

@Praful932 @maneprajakta I have made 2 different prototypes for small screens. Pls check them and would love some feedback on the same. https://www.figma.com/file/quY3GKiD5RgWhsJAg2PKyE/Carousel-draft-Kitaabe?node-id=0%3A1

@Praful932 Okay so I'll implement auto-scroll for the 1st one. But regarding this, on one scroll, should I let it move forward by 1 block/ all 4 blocks?

**Comment by maneprajakta** [comment]
@Greeshma2903 yep it's not like that 😅 we had arranged alphabetically in groups.
All three designs are Great 💫 but **carousel each with 3 blocks** seems perfect to me.

**Comment by Greeshma2903** [comment]
> @Greeshma2903 yep it's not like that 😅 we had arranged alphabetically in groups.
> All three designs are Great 💫 but **carousel each with 3 blocks** seems perfect to me.

Oh 😅
thank you! @maneprajakta I have enabled editing for you if you want to make particular changes in UI. should I now clone the project and start with the code part? OR should I start after you make the changes?

**Comment by maneprajakta** [comment]
@Greeshma2903 No changes from my side you can start working on it.
Yes, Bootstrap 4 is used.
Go through [Starter Guide](https://github.com/Praful932/Kitabe/discussions/42) once it will help you in setting up the project.

**Comment by Praful932** [comment]
> Um, actually I thought there were multiple carousels, so each carousel might be a group of similar genres (like fiction and its sub-genres). But that wasn't the case 😅.
> 
> @Praful932 @maneprajakta I have made 2 different prototypes for small screens. Pls check them and would love some feedback on the same. https://www.figma.com/file/quY3GKiD5RgWhsJAg2PKyE/Carousel-draft-Kitaabe?node-id=0%3A1
> 
> @Praful932 Okay so I'll implement auto-scroll for the 1st one. But regarding this, on one scroll, should I let it move forward by 1 block/ all 4 blocks?

Mobile view seems good to me. **Let it move forward by 1 block** You can start working on it :)
Happy hacking!

**Comment by Greeshma2903** [comment]
Great! 
@maneprajakta I had a query regarding the steps after cloning! Since I am contributing only to the frontend part, do I still need to install virtualenv? can I just create a new branch and start fixing the issue?

**Comment by maneprajakta** [comment]
@Greeshma2903 
Yes as you need to view the changes you made locally right. If you don't set up a project like installing everything from requirements.txt the project won't run. You will not be able to view the changes done.

**Comment by Greeshma2903** [comment]
okay thanks!

**Comment by Greeshma2903** [comment]
Hey, when I tried installing virtualenv, it shows:
`bash: pip: command not found`

Do I need to download python env first?

**Comment by maneprajakta** [comment]
@Greeshma2903 Have you installed **pip** ?
Also what OS you are using.
No issues :)

**Comment by Greeshma2903** [comment]
> @Greeshma2903 Have you installed **pip** ?
> Also what OS you are using.
> No issues :)

Windows
Nope, didn't know about **pip**, will do so

**Comment by Greeshma2903** [comment]
> > @Greeshma2903 Have you installed **pip** ?
> > Also what OS you are using.
> > No issues :)
> 
> Windows
> Nope, didn't know about **pip**, will do so

**Comment by Greeshma2903** [comment]
Sorry by mistake closed the issue😶

**Comment by Greeshma2903** [comment]
I think **python** has not come pre-installed on my OS. I was checking a guide to install it and realized that different versions of python have different requirements. Which one will I require for this project? Python 2.7 OR 3.9?

**Comment by maneprajakta** [comment]
Yep, in windows, it doesn't come pre-installed. Go with the latest version of python.

**Comment by Praful932** [comment]
> The Project works seamlessly on Python version 3.8.6

[CONTRIBUTING.md](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md)

**Comment by Greeshma2903** [comment]
Thanks!

**Comment by Greeshma2903** [comment]
@Praful932 can I dm you the SS on discord, the env is not getting activated

**Comment by maneprajakta** [comment]
@Greeshma2903 any updates?

**Comment by Greeshma2903** [comment]
I would like to apologize for the delay (due to exams), but work is in progress😅
I made some changes, and the carousel now works something like this (shown in video)
https://www.loom.com/share/3ddc356773a14b73a5aa8e4de8c2ea50
I was not able to figure out how I can move the carousel forward by 1 block (Since Bootstrap doesn't provide a multi-image sliding option), so decided on doing 3 blocks at a time.

I am presently working on the carousel for the mobile view. But I realised since the text is in the picture if I reduce the height (for small screens), the text won't be visible to us.  Presently looks like this with block-sized pictures:
https://www.loom.com/share/87d9979b88944854a38eb94faea7264e
(Pls ignore the positioning of controls, WIP😅)
@maneprajakta
